mobile-app/app/api/persistence/custom_slippage-tolerance_storage.ts

```diff
@@ -0,0 +1,18 @@
+import AsyncStorage from "@react-native-async-storage/async-storage";
+
+const KEY = "WALLET.CUSTOM_SLIPPAGE_TOLERANCE";
+
+async function set(
+  customSlippageTolerance: NonNullable<string>
+): Promise<void> {
+  await AsyncStorage.setItem(KEY, customSlippageTolerance);
+}
+async function get(): Promise<string> {
```

Comment:
can't we use the existing one?

Reply:
This is to store the true/false value of whether a custom value is used or not, not the actual value of the slippage tolerance input value

Reply:
Yes but if I'm not mistaken, the old logic does it as well. If there's a value, then use custom. Otherwise, not. Would it work similarly?

Reply:
Will remove the custom storage and handle it on the front instead